File: javascript/ArithmeticApp.js

    const prompt = require("prompt-sync")();    
       

        let trial = 0;
        let score =0;
        let firstNumber =0;
        let secondNumber = 0;
        let result = 0;
        let range  = 10;

        let startTime = Date.now();
        
    console.log("Time Start Now:");
    
        
    console.log("Solve Mathematical problems\nOn differences of two Numbers\n");
    while(trial != range){
         
            firstNumber = Math.floor(Math.random(52)* 23);
            secondNumber = Math.floor(Math.random(13) * 12);

         if(firstNumber > secondNumber){
            console.log((trial+1),  ". ",firstNumber , " - ", secondNumber+ " = ");
            result = prompt("   Enter result: ")

                if(result != firstNumber - secondNumber){          
                    console.log("  Try again");
                    console.log("  ", firstNumber, " - ", secondNumber, " = ");
                    let  result = prompt("    Enter result: ")
                        if(result == firstNumber - secondNumber){ 
                            score++; 
                            console.log("  correct");
                         }
               }
               else if(result == firstNumber - secondNumber){ 
                    score++; 
                    console.log("   correct");
                }
                    trial++;
         }
    }
    let stopTime = Date.now()
    let total = startTime - stopTime
        total =Math.floor(total /-1000); 
    console.log("Total time used(sec):", total);
    console.log("Score: ", score ," of ", range, " questions");
    

    

        

File: python/ArithmeticApp.py
import random
import time
import math

trial = 0;
score =0;
first_number =0;
second_number = 0;
result = 0;
time_range  = 10;

start_time = time.time();
        
print("Time Start Now");
    
print("Solve Mathematical problems\nOn differences of two Numbers\n");
while trial != time_range:
         
    first_number = random.randint(15,30);
    second_number = random.randint(2, 14);

    print((trial+1),  ". ",first_number , " - ", second_number, " = ");
    result = int(input("   Enter result: "))

    if result != first_number - second_number:        
        print("  Try again");
        print("  ", first_number, " - ", second_number, " = ");
        result = int(input("    Enter result: "))

        if result == first_number - second_number:
            score+=1; 
            print("  correct");

    elif result == first_number - second_number: 
        score+=1; 
        print(" correct");
    trial+=1;
    
stop_time = time.time();
total = math.floor(stop_time - start_time);
print("Total time used(sec): ", total);
print("Score: ", score ," of ", time_range, " questions");
    

    

        







